throw error if nothing to inject

tiberiuzuld opened the pull request.

joakimbeng commented on src/inject/index.js:
```diff
@@ -110,7 +110,7 @@ function getNewContent(target, collection, opt) {
       log(cyan(filesCount) + ' file' + pluralState + ' into ' + magenta(target.relative) + '.');
     } else {
-      log('Nothing to inject into ' + magenta(target.relative) + '.');
+      error('Nothing to inject into ' + magenta(target.relative) + '.');
```
You'll need a `throw` here as well, otherwise nothing will happen, it won't even log anything...

tiberiuzuld replied:
Done

joakimbeng commented on src/inject/index.js:
```diff
@@ -110,7 +110,7 @@ function getNewContent(target, collection, opt) {
       log(cyan(filesCount) + ' file' + pluralState + ' into ' + magenta(target.relative) + '.');
     } else {
-      log('Nothing to inject into ' + magenta(target.relative) + '.');
+      error('Nothing to inject into ' + magenta(target.relative) + '.');
```
I also suggest adding a test for this

tiberiuzuld replied:
Added tests for both error and log if there is nothing to inject

tiberiuzuld commented:
Hi @joakimbeng ,
Added extra option `throwErrorIfNoInject` default `false` for this feature.
Added tests for log and error.
Changed the commit message to `feat: throw error if nothing to inject`

Please review when you have time.

Thanks

buberdds commented:
Hey @joakimbeng, is there a chance to merge this PR?
